config/crds/tenancy.kcp.dev_clusterworkspacetypes.yaml

```diff
@@ -44,6 +44,21 @@ spec:
                 description: additionalWorkspaceLabels are a set of labels that will
                   be added to a ClusterWorkspace on creation.
                 type: object
+              allowedSubWorkspaceTypes:
+                description: "allowedSubWorkspaceTypes is a list of ClusterWorkspaceTypes
+                  that can be created in a workspace of this type. \n By default,
+                  no type is allowed. This means no other workspace can be nested
+                  within a workspace of the given type. The name `*` allows any child
+                  type to be nested."
```

Comment:
should we use child type or sub-workspace-type as the wording. We should unify. My gut feeling: sub-type has the conotation of type theory.

Reply:
nit: should we default this field to empty list? I think that would make it clear. Omitting the field and then forbidden something is odd.

Reply:
unresolved

Reply:
Try in tenancy.kcp.dev_clusterworkspacetype.yaml-patch:
```
- op: add
  path: /spec/versions/name=v1alpha1/schema/openAPIV3Schema/properties/spec/properties/allowedChildWorkspaceTypes/default
  value: []
```

Reply:
We don't have to technically. But it is very bad UX as it is now as it is hard to discover that no child type is allowed. A default value of `[]` makes it visible to everybody.